app/templates/admin/userManagement.html

```diff
@@ -19,48 +17,109 @@ <h1 class="text-center mt-5 mb-5">Admin Management</h1>
+              </div>
+            </div>
           </div>
         </div>
-        <button class="btn btn-primary my-2 float-end" id="submitButton" role="button">Make Current Term</button>
+      </div>
+  </div>
+  <div class="accordion-item">
+    <h3 class="accordion-header" id="headingTwo">
+      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+          Term Management Settings
+        </button>
+        </h3>
+        <div id="collapseTwo" class="accordion-collapse collapse" aria-labelledby="headingTwo" data-bs-parent="#adminManagement">
+          <div class="accordion-body">
+              <div class="row">
+                <div class="container-fluid mt-5 col-md-4">
+                  <div class="text-center">
+                    <div class="card">
+                      <div class="card-header"><strong>Select the Current Term</strong></div>
+                        <ul class="list-group" name="currentTermList" id="currentTermList">
+                        {% for term in terms %}
+                        <li class="term-btn list-group-item list-group-item-action {{ 'active' if g.current_term == term }}"
+                          id="termFormID_{{ loop.index }}"
+                          aria-label="{{'Current term' if g.current_term == term}}"
+                          value='{{ term.id }}'
+                          tabindex="0">
+                          {{term.description}}
+                          <span class="sr-only visually-hidden">{{'Selected term' if g.current_term == term}}</span>
+                      </li>
+                      {% endfor %}
+                      </ul>
+                      <div class="card-footer">
+                        <button class="btn btn-success w-100" id="addNewTerm" role ="button">Add New Term</button>
+                      </div>
+                    </div>
+                    <button class="btn btn-primary my-2 float-end" id="submitButton" role="button">Make Current Term</button>
+                  </div>
+              </div>
+          </div>
       </div>
     </div>
+  </div>
+  <div class="accordion-item">
+    <h3 class="accordion-header" id="headingThree">
+      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
+        Program Settings
+        </button>
+      </h3>
+      <div id="collapseThree" class="accordion-collapse collapse" aria-labelledby="headingThree" data-bs-parent="#adminManagement">
+        <div class="accordion-body">
+          <table>
+            <tr>
+              <th>Program Select</th>
+              <th>Reply-To-Email</th>
+              <th>Sender Name</th>
+            </tr>
+            <tr>
+            <form action="" id="programSettings">
+                      <div>
```

Comment:
The div and form tags shouldn't be in the table row (but outside of the table cell). Definitely for the div, but the form feels weird as well